misk-hibernate/src/main/kotlin/misk/hibernate/migrate/BulkShardMigrator.kt

```diff
@@ -186,7 +186,7 @@ class BulkShardMigrator<R : DbRoot<R>, C : DbChild<R, C>> private constructor(
           val update = """
           UPDATE ${tableName()}
           SET $setColumns
-          WHERE $rootColumnName = ?
+          WHERE $where
```

Comment:
```where``` clause already contains the predicate to filter child entities from the source root. This is done in ```BulkShardMigrator.source(...)```